samples/cellular/nrf_cloud_multi_service/sysbuild/mcuboot/boards/thingy91x_nrf9151.conf

```diff
@@ -14,3 +14,8 @@ CONFIG_UART_CONSOLE=n
 CONFIG_MCUBOOT_SERIAL=y
 CONFIG_MCUBOOT_SERIAL_DIRECT_IMAGE_UPLOAD=y
 CONFIG_BOOT_SERIAL_IMG_GRP_IMAGE_STATE=y
+
```

Comment:
Seems ok. @tony-le-24 can you check if the various additional nightly and weekly tests still work with this change -- any of them that run on the Thingy:91X?

Reply:
I will, thanks

Reply:
I ran it on my PR for Thingy91:X testing, things seem green. LGTM!